update EyeDropper API (#29858)
mdn/content: files/en-us/web/api/eyedropper/open/index.md

### Exceptions

Exceptions are not thrown but returned when the {{jsxref("Promise")}} is rejected.

- `NotAllowedError` {{domxref("DOMException")}}

>   - : Thrown if it is not called via a [transient user activation](/en-US/docs/Glossary/Transient_activation).

Reviewer: ```suggestion
  - : Thrown if `open()` is not called via a [transient user activation](/en-US/docs/Glossary/Transient_activation).
```

- `InvalidStateError` {{domxref("DOMException")}}
  - : Thrown if another eye dropper has already opened.
- `AbortError` {{domxref("DOMException")}}

>   - : Thrown if the user abort the selection by pressing the <kbd>Escape</kbd> key or is aborted by the {{domxref("AbortController")}} passed as an argument to `open()`.

Reviewer: ```suggestion
  - : Thrown if the user aborts the selection by pressing the <kbd>Escape</kbd> key or if the selection is aborted by an {{domxref("AbortController")}} being passed as an argument to `open()`.
```

>   - : Thrown if the user aborts the selection by pressing the <kbd>Escape</kbd> key or if the selection is aborted by an {{domxref("AbortController")}} being passed as an argument to `open()`.

Reviewer: ```suggestion
  - : Thrown if the user aborts the selection by pressing the <kbd>Esc</kbd> key or if the selection is aborted by an {{domxref("AbortController")}} being passed as an argument to `open()`.
```
